Makefile

```diff
@@ -265,28 +265,6 @@ migrate:
 dbchange:
 	$(MANAGEMENT_COMMAND) makemigrations
 
-server_noattach:
```

Comment:
Did you make sure nobody on the eng team is still using this target?  I wouldn't think so, given that we're now running everything in `supervisord`, which is much more flexible.

Reply:
Hey, I still use this occasionally :(